src/assets/styles/layout.css

```diff
@@ -1,15 +1,204 @@
-body {
+:root {
+  --color-white: rgb(255, 255, 255);
+  --color-black: rgb(0, 0, 0);
+}
+
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body,
+html {
   height: 100%;
   width: 100%;
   overflow-x: hidden;
   margin: 0;
+  padding: 0;
 }
 
-.full-div {
-  background-image: url(../img/slides/slide-1.jpg);
-  background-size: contain;
-  background-repeat: no-repeat;
-  background-position: center;
-  height: 100vh;
+#root {
+  height: 100%;
+  width: 100%;
+  margin: 0;
+}
+
+body::-webkit-scrollbar {
+  width: 1em;
+}
+ 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=5731031)